<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gobang</class>
 <widget class="QWidget" name="gobang">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_return">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_timeuse">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>300</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>ai计算耗时:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_nodeNum">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>330</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>ai叶结点数:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>51</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>白棋</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd_write"/>
    </item>
    <item>
     <widget class="QLabel" name="lb_white_position">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>190</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>黑棋 </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd_black"/>
    </item>
    <item>
     <widget class="QLabel" name="lb_black_position">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lb_eval">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>360</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>ai局面估分:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
